resources/assets/coffee/react/beatmapset-page/scoreboard-table-row.coffee

```diff
@@ -53,34 +56,49 @@ export class ScoreboardTableRow extends React.PureComponent
 
       td className: cell,
         a
-          className: "#{bn}__user-link js-usercard"
+          className: "#{bn}__cell-content #{bn}__cell-content--user-link js-usercard"
```

Comment:
the whole column now links to user. Not sure which one is better.

![](https://s.myconan.net/2020-11/msedge_2020-11-20_16-46-55.png)

Reply:
The top score one already behaves this way, this makes it behave the same way.